### PR TITLE
blog: Draft of Miro playground piece

### PR DESCRIPTION
**Pull request by jendefig**
This is in draft form.  Any reviews at the moment are for poking holes in the idea in general.  Still lots of work to do on specifics and editing in writing.

The Miro board in its current state live here: https://miro.com/app/board/o9J_lu51CE0=/?moveToWidget=3074457366373673738&cot=14

It needs more:
- Components to work with
- requests for diagrams from the community
- variation on type of diagrams (small and specific to large and general)

### CODE REVIEW

File: content/blog/2021-11-23-mlops-architecture-playground.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+- The best (and most shared) content has good images and diagrams to drive
+  clarity
+- Given the flexibility and hands-off-your-code characteristics of our tools, it
+  can sometimes be challenging for people to conceptualize their workflows
+- diagram images can often be hard to read in a blog post because they take up a
+  lot of space and are illegible when condensed
+- the best solutions and ideas most often come from many places
+- many hands make light work
+- we have an awesome Community of exceptional humans!
```
Comment (dmpetrov) [inline content/blog/2021-11-23-mlops-architecture-playground.md:40]: Lower/Capital latter in the first word

File: content/blog/2021-11-23-mlops-architecture-playground.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+board was used to collaborate and solve problems, I came up with the idea for
+the **MLOps Architecture Playground** in Miro.
+
+![MLOps Architecture Playground by Iterative.ai](/uploads/images/2021-11-23/mlops-playground.png)
+_Join us in the MLOps Archtecture Playground!
+([Source link](https://www.ravirajag.dev/blog/mlops-summary))_
+
+### What?
+
+What this will be is a place for people in the greater MLOps Community to come
```
Comment (dmpetrov) [inline content/blog/2021-11-23-mlops-architecture-playground.md:52]: In general, I'd refer to ML engineers or data scientists rather than MLOps Community.

Comment (dmpetrov) [reply content/blog/2021-11-23-mlops-architecture-playground.md:52]: I bit more details. These two are our target audience. I don't know how happy these folks would be if you call them MLOps. They might decide the tool is not for them.

File: content/blog/2021-11-23-mlops-architecture-playground.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+- many hands make light work
+- we have an awesome Community of exceptional humans!
+
+So digesting these things and attending other community events where a Miro
+board was used to collaborate and solve problems, I came up with the idea for
+the **MLOps Architecture Playground** in Miro.
+
+![MLOps Architecture Playground by Iterative.ai](/uploads/images/2021-11-23/mlops-playground.png)
+_Join us in the MLOps Archtecture Playground!
+([Source link](https://www.ravirajag.dev/blog/mlops-summary))_
```
Comment (dmpetrov) [inline content/blog/2021-11-23-mlops-architecture-playground.md:48]: I have not find the image in this source, Might be I'm missing something.

File: content/blog/2021-11-23-mlops-architecture-playground.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+### What?
+
+What this will be is a place for people in the greater MLOps Community to come
+and work out their architecture issues. This could be a small piece of the
+puzzle or the greater architecture of your entire workflow and tool stack.  
+What we aim to provide you is a place to work it out and all the components you
+need to create it fast (or as fast as possible, given that this is complicated
+stuff!). Your work in here will likely help other Community members struggling
+with the same issues.
```
Comment (dmpetrov) [inline content/blog/2021-11-23-mlops-architecture-playground.md:58]: Any examples of struggles? It still looks a bit abstract to me 😄

File: content/blog/2021-11-23-mlops-architecture-playground.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+![MLOps Architecture Playground by Iterative.ai](/uploads/images/2021-11-23/mlops-playground.png)
+_Join us in the MLOps Archtecture Playground!
+([Source link](https://www.ravirajag.dev/blog/mlops-summary))_
+
+### What?
+
+What this will be is a place for people in the greater MLOps Community to come
+and work out their architecture issues. This could be a small piece of the
+puzzle or the greater architecture of your entire workflow and tool stack.  
+What we aim to provide you is a place to work it out and all the components you
```
Comment (dmpetrov) [inline content/blog/2021-11-23-mlops-architecture-playground.md:55]: Is it an issue with new line / formatting?

File: content/blog/2021-11-23-mlops-architecture-playground.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+3. Make a copy of the Label and fill out the items on it as they pertain to your
+   diagram. Include your contact info (email, Discord, LinkedIn, Twitter).
+   Whatever you are comfortable with. You never know when you will find a new
+   friend somewhere around the world that is grappling with a similar thing and
+   you can work it out together.
+4. Make copies of the different components and start creating your diagram!
+5. Checkout the Iterative tools request board - grab a request and get to work!
+   Anyone that has completed one of the desired diagrams will get some swag! 🤩
+   Email Jeny@iterative.ai when complete with a link to the diagram.
```
Comment (dmpetrov) [inline content/blog/2021-11-23-mlops-architecture-playground.md:90]: email in text? You will be getting more spam 😄

File: content/blog/2021-11-23-mlops-architecture-playground.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+### Who is this board for?
+
+Everyone! You don't have to use only Iterative tools to use this board. We know
+that our tools are good building blocks in your MLOps tools stack, but there are
+many other tools. And our tools integrate with many. Feel free to work out your
+diagram even if it does not use our tools. We're all in this together!
+
+### How will this work?
+
+1. Use this link [needed from MailChimp, figuring out], to enter the MLOps
```
Comment (dmpetrov) [inline content/blog/2021-11-23-mlops-architecture-playground.md:71]: Do I need Miro account?

**Review by dmpetrov — CHANGES_REQUESTED** [review]
@jendefig great initiative! Thank you for the idea and driving this.

I wish we have more use cases to demonstrate full power of such tools.

File: content/blog/2021-11-23-mlops-architecture-playground.md (hunk 7 above)
Comment (jendefig) [reply content/blog/2021-11-23-mlops-architecture-playground.md:71]: No you don't need a Miro account to join and can still edit the board.  Just the person who hosts needs a Miro account. ✅

File: content/blog/2021-11-23-mlops-architecture-playground.md (hunk 3 above)
Comment (jendefig) [reply content/blog/2021-11-23-mlops-architecture-playground.md:48]: It's not there yet.  Still in draft form.

**Comment by shcheklein** [comment]
@jendefig any updates here? or should we close this?

**Comment by jendefig** [comment]
> @jendefig any updates here? or should we close this?

This is not dead.  @mnrozhkov and I and team need to work on POC more before we release to the world.

**Comment by shcheklein** [comment]
@jendefig any updates by chance. ETA on this? :)

**Comment by jendefig** [comment]
> @jendefig any updates by chance. ETA on this? :)

End of Q2

**Comment by shcheklein** [comment]
@jendefig can we close the PR for now? (it looks better when we don't grow the number of open PRs that are still in process)

**Comment by jendefig** [comment]
Closing, for now, to revisit with @mnrozhkov in Q2.

**Pull request closed without merging**